scripts/linters/pre_commit_linter.py

```diff
@@ -428,19 +428,41 @@ def _print_complete_summary_of_errors(all_messages):
             python_utils.PRINT(message)
 
 
-def _get_task_output(all_messages, task, semaphore):
+def _get_task_output(all_messages, err_messages, task, semaphore):
     """Returns output of running tasks.
 
     Args:
         all_messages: list(str). List of summary messages of linter output.
+        err_messages: list(str). List of error messages during linter execution.
```

Comment:
I'm not sure about the naming here, all_message will contain success and failure messages, right? is there any chance of all_messgaes to contain lint error message as well (not failure or exception messages)?

We should find a better way to name this either make all_message to be a dict with success, failure and error keys. or find a better name for the `all_messages`. (maybe lint_messages?)

See line 421 also deals with error message which is confusing*

Reply:
I am planning to add lint error messages as well in a later PR. As of now we only have Success and Failure messages in it.

Changed the all_messages to lint_messages.